scripts/validation/validate-versions-staged-files.js

```diff
@@ -22,10 +22,10 @@ async function precommit() {
             staged.some(
                 changedFilePath =>
                     dirname(join(process.cwd(), changedFilePath)).includes(location) &&
-                    ["package.json", "package.xml"].includes(changedFilePath.split("/").pop())
+                    new RegExp(/(package.json)|(package.xml)/).test(changedFilePath.split("/").pop())
```

Comment:
You dont need to use RegExp

Reply:
This should solve your problem
```suggestion
                    changedFilePath.split("/").pop()?.match(/package.(json|xml)$/)
```

Reply:
Note that the `.` should be escaped with a backslash when used as a literal in a regex, or a string like `packagesjson` would also match :).

Reply:
Changed my suggestion :P